src/main/groovy/com/vherasymenko/avro/encoder/core/AvroEncoderPort.groovy

```diff
@@ -0,0 +1,11 @@
+package com.vherasymenko.avro.encoder.core
+
+import org.apache.avro.Schema
+
+/**
+ * The avro encoder port.
+ */
+interface AvroEncoderPort {
+
+    String encodeEvent(Schema schema, def avroData, Class classType )
```

Comment:
I want to use this encoder for different documents. That's why I used `def` to not defined the type of `avroData` here.